src/components/Button/ButtonSpinner.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Spinner } from "../Spinner";
+
+import "./styles.css";
+
+type ButtonSpinnerProps = {
+  label?: string;
+};
+
+export const ButtonSpinner: React.FC<ButtonSpinnerProps> = ({ label }) => (
```

Comment:
Oh, let me make this prop clearer. It would be easier as a boolean. Basically using it to know if we need to set some margin, or if the spinner should be centered.